src/main.cpp

```diff
@@ -98,6 +100,10 @@ int main()
     Logger::logger.log(Logger::Severity::Information, "Start of program");
     std::cout << localization::Ressources::rsHello << std::endl;
 
+    /// setup audio for program
+    raylib::core::Audio audio;
+    audio.initDevice();
```

Comment:
Create a new Scoped class instead of doing initDevice/closeDevice.
See raylib/core/scoped.hpp.